src/lib/descriptor/makefiles/rust.toml

```diff
@@ -733,17 +733,40 @@ run_task = "do-copy-apidocs"
 description = "Copies the generated documentation to the docs/api directory."
 category = "Documentation"
 private = true
-script_runner = "@shell"
+script_runner = "@duckscript"
 script = [
 '''
-SOURCE_DIRECTORY=${CARGO_MAKE_DOCS_ROOT_FOLDER}/target/doc
-TARGET_DIRECTORY=${CARGO_MAKE_DOCS_ROOT_FOLDER}/docs/api/${CARGO_MAKE_DOCS_SUB_FOLDER}
+if is_empty ${CARGO_MAKE_DOCS_SUB_FOLDER}
+    SOURCE_DIRECTORY= set ${CARGO_MAKE_DOCS_ROOT_FOLDER}
```

Comment:
since SOURCE_DIRECTORY is now actually a variable, can we name it source_dir or something short?
same goes for other variables.

Reply:
ok